src/model/clients/GenericResponse.js

```diff
@@ -103,4 +103,26 @@ export class GenericResponse {
   get headers() {
     return Object.assign({}, this.config.headers);
   }
+
+  /**
+   * Convert the object to a `Response` object returned by the fetch command.
+   * @returns {Response}
+   */
+  convertToFetchResponse() {
```

Comment:
This converts our `GenericResponse` to a `Response` object, which is used by the global `fetch` method.